tests/EA/test_basic_island.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import numpy as np
+
+from bingo.MultipleValues import SinglePointCrossover, SinglePointMutation, MultipleValueGenerator
+from bingo.EA.VarOr import VarOr
+from bingo.Island import Island
+from bingo.EA.MuPlusLambda import MuPlusLambda
+from bingo.EA.TournamentSelection import Tournament
+from bingo.EA.SimpleEvaluation import SimpleEvaluation
+from examples.OneMaxExample import MultipleValueFitnessEvaluator
+
+@pytest.fixture
+def island():
```

Comment:
lets add a tests here for checking popper generational age and genetic age increases